.github/workflows/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,140 @@
+name: Bug report
+description: Create a report to help us fix something within the Windows Community Toolkit.
+labels: ["bug :bug:"]
+body:
+- type: markdown
+  attributes: 
+    value: |
+      ## Before you begin
+      🚨 **Please do not skip instructions**🚨
+      
+      This info is essential for investigating your bug report. Issues with missing information may be closed without investigation.
+      
+      If you're uncertain about a problem, find or start a [Questions & Answers](https://github.com/CommunityToolkit/Windows/discussions/categories/q-a) discussion, where you can get insight from the community regarding your issue.       
+- type: textarea
+  id: description
+  validations:
+    required: true
+  attributes:
+     label: Describe the bug
+     description: A clear and concise description of what the bug is.
+- type: textarea
+  id: repro-steps
+  validations:
+    required: true
+  attributes:
+     label: Steps to reproduce
+     render: text
+     description: Provide steps to reproduce the issue, or let us know why it can't be reproduced (e.g. more complex setup, environment, dependencies, etc...)
+     placeholder: |
+        A repro from the sample app expedites the process of discovering, resolving, and validating bug fixes. 
```

Comment:
Should we link to the template? https://github.com/michael-hawker/WinUI2Plus3Template

Reply:
The dependencies in this template aren't kept in step with the Toolkit. The majority of bugs should repro in a component sample or test within our repo, so I would hold onto this until the cases where it's needed.

Reply:
I mean I guess the idea is to provide a pointer to a template where they can produce a minimal repro, and call out that bugs with an isolated repro are easier for folks to fix (and for us to hand-off to the platform folks). Maybe we need a dedicated template for this? Or some other instructions?

Reply:
I agree. The process is much simpler now that components are easy to create and throw away, but no doubt users will need some basic "Getting Started" instructions.

Reply:
I mean I guess the specific guidance here was for more when we had the XAML Live Preview in the old app, so probably something more like:

> A solid list of repro steps enables others or maintainers to understand how to recreate your issue and enable validating fixes and resolving the problem.
> It is best to provide a stand-alone project that shows how to reproduce the issue with as few steps as possible.
> This makes it easier to understand where the issue may be without having to understand complexities of systems unique to your application.
> If the problem can be reproduced in our Sample Gallery (https://aka.ms/windowstoolkitapp), then provide those instructions here instead:

Example repo steps...